package.json

```diff
@@ -9,9 +9,9 @@
     "tdd": "cross-env NODE_ENV=unittest jest --watch",
     "prebuild": "npm run clean",
     "build": "cross-env NODE_ENV=production webpack --config ./webpack.config.prod.js --bail",
-    "start": "cross-env NODE_ENV=development DISABLE_SSR=true nodemon server --watch server",
-    "start-with-ssr": "cross-env NODE_ENV=development nodemon server --watch server",
-    "start-prod": "cross-env NODE_ENV=production DISABLE_SSR=true node server",
+    "start": "cross-env NODE_ENV=unittest DISABLE_SSR=true nodemon server --watch server",
+    "start-with-ssr": "cross-env NODE_ENV=unittest nodemon server --watch server",
```

Comment:
Hvorfor endrer vi disse til unittest?

Reply:
Ingen praktisk betydning, men heng sammen med endringa i config som er henta fra ein av dei andre applikasjonene her. Eg har også lagt på BUILD_TARGET som et hack for å trigge koden i config. Denne variabelen settes av razzle så den kan tas bort når du er ferdig med det. Det gjør at når koden kjøres på server eller lokalt så hentes config fra config, mens i nettleser så hentes den fra window.config, som settes i Html.jsx.